integration/testutil.go

```diff
@@ -304,7 +304,8 @@ func dockerhub(name string, opts ...imageOpt) imageInfo {
 	return i
 }
 
-func encodeImageInfo(ii ...imageInfo) [][]string {
+// encodeImageInfoCtr assembles command line options for pulling or pushing an image using ctr
+func encodeImageInfoCtr(ii ...imageInfo) [][]string {
```

Comment:
This function is no longer used but I predict it will be a useful helper when we need to explicitly call `ctr` in the future for features not present in `nerdctl`.